### PR TITLE
C4C-166 Order Items

### PR DESCRIPTION
**Pull request by hackeret**
- Fixed pushing order items to database

- Fixed the updating of item quantity when items ordered

- Updated tests to use asserts

### CODE REVIEW

File: C4CApp/C4CWebTests/DatabaseConnectorTests.cs [hunk 1]
```diff
@@ -198,27 +215,38 @@ public void TestDeletePeople()
...
-        //    DatabaseConnector.Push_Order_Items_To_Database(orders);
-        //    DatabaseConnector.DeleteOrderItems(orders);
+        [TestMethod]
+        [ExpectedException(typeof(IndexOutOfRangeException), "There is no row at position 0")]
+        public void TestDeleteOrderItems()
+        {
+            DatabaseConnector.RefreshData();
+            DatabaseConnector.Push_Orders_to_database(orders);
+            DatabaseConnector.Push_Order_Items_To_Database(orders);
+            DatabaseConnector.Update_Item_Quantities(orders);
```
Comment (rekowp17) [inline C4CApp/C4CWebTests/DatabaseConnectorTests.cs]: Can you change these method names to reflect coding conventions?

File: C4CApp/C4CWebTests/DatabaseConnectorTests.cs [hunk 2]
```diff
@@ -113,36 +117,43 @@ public void TestPushOrdersToDatabase()
...
-        //[TestMethod]
-        //public void TestPushOrdersItemsToDatabase()
-        //{
-        //    DatabaseConnector.RefreshData();
-        //    DatabaseConnector.Push_Order_Items_To_Database(orders);
+        [TestMethod]
+        public void TestPushOrdersItemsToDatabase()
+        {
+            DatabaseConnector.RefreshData();
+            DatabaseConnector.Push_Orders_to_database(orders);
```
Comment (rekowp17) [inline C4CApp/C4CWebTests/DatabaseConnectorTests.cs]: Can you change these method names to reflect coding conventions?

File: C4CApp/C4CWeb/Controllers/OrderSubmissionController.cs [hunk 3]
```diff
@@ -335,6 +335,8 @@ protected void updateDatabase(Order order)
             people[1] = order.Recipient; 
             DatabaseConnector.Push_Orders_to_database(orders);
             DatabaseConnector.Push_People_To_Database(people);
```
Comment (rekowp17) [inline C4CApp/C4CWeb/Controllers/OrderSubmissionController.cs]: Can you change these method names to reflect coding conventions?

File: C4CApp/C4CWeb/Controllers/OrderSubmissionController.cs [hunk 4]
```diff
@@ -335,6 +335,12 @@ protected void UpdateDatabase(Order order)
             people[1] = order.recipient; 
             DatabaseConnector.PushOrdersToDatabase(orders);
             DatabaseConnector.PushPeopleToDatabase(people);
+            DatabaseConnector.PushOrderItemsToDatabase(orders);
+            DatabaseConnector.UpdateItemQuantities(orders);
```
Comment (kubowitsche) [inline C4CApp/C4CWeb/Controllers/OrderSubmissionController.cs]: Is there a reason we push order to database twice? you have it once on 336 and once on 338

Comment (kubowitsche) [inline C4CApp/C4CWeb/Controllers/OrderSubmissionController.cs]: Just kidding. Can't read.

**Pull request merged**